scripts/page_browse.js

```diff
@@ -336,16 +331,15 @@ function pageBrowse(params, storageKey, replaceUrl, mentorMode = false) {
                 resolve();
             } else {
                 roundSelector = document.createElement("select");
-                $.ajax(makeApiAjaxSettings("v1/projects/pagerounds"))
-                    .done(function(rounds) {
+                ajax("GET", "v1/projects/pagerounds")
+                    .then(function(rounds) {
```

Comment:
if you want, we could use async await, but probably drops support a bit more.

Reply:
Yes async and await could simplify the ajax function as well. They are supported in firefox 52 and safari 11, not chrome 54 or opera 41.